README.md

```diff
@@ -24,9 +24,101 @@ system-under-test that is expected to cause a change to some output(s).
 
 ## Installation
 
-See the Read the Docs site for [installation
-instructions](https://causal-testing-framework.readthedocs.io/en/latest/installation.html).
+### Requirements
+- Python >= 3.9.
```

Comment:
I'm not confident we support 3.11 or 3.12 yet, so it may be worth being more specific until we know we can. I'm working on merging in a branch that will add all python versions to the test matrix for the GH CI action.